lib/Math/Business/LookBacks.pm

```diff
@@ -0,0 +1,277 @@
+package Math::Business::LookBacks;
```

Comment:
I see a mixed of naming. In contract category definition it is called `lookback` and here it is `lookbacks`. Choose one. I think `lookback` is more correct.